op-batcher/batcher/channel_builder_test.go

```diff
@@ -758,6 +765,28 @@ func TestFramePublished(t *testing.T) {
 	require.Equal(t, uint64(1000), cb.timeout)
 }
 
+func TestLatestL1Origin(t *testing.T) {
```

Comment:
```suggestion
func TestChannelBuilder_LatestL1Origin(t *testing.T) {
```
I know this convention is not applied everywhere else...